CMakeLists.txt

```diff
@@ -201,6 +211,14 @@ add_dependencies(psi4_core ambit_external
                            libint_external
                            pcmsolver_external
                            pybind11_external)
+if(ENABLE_psi4pasture)
+  ExternalProject_Add(psi4pasture
+    GIT_REPOSITORY https://github.com/amjames2/psi4pasture
```

Comment:
I think this if/endif should go in a `external/pasture/CMakeLists.txt`. I'm fine with keeping it in `external` folder unless ppl really want an `internal` folder.